### PR TITLE
Session Info API endpoint in Ecommerce for Edly Panel

### PR DESCRIPTION
**Pull request by asadmanzoor93**
**Description:** This PR adds session info API endpoint in Ecommerce for Edly Panel

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-1026

**Merge checklist:**

- [ ] All reviewers approved
- [ ] Commits are (reasonably) squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable to change.

### CODE REVIEW

File: ecommerce/extensions/edly_ecommerce_app/api/v1/tests/test_views.py [hunk 1]
```diff
@@ -64,3 +64,37 @@ def test_update_current_site_theme(self):
...
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_get_user_session_info(self):
+        """
+        Verify response on list view.
+        """
+        response = self.client.get(self.session_url, format='json')
+
+        assert response.status_code == status.HTTP_200_OK
+        assert "csrf_token" in response.json()
```
Comment (zubair-arbi) [inline ecommerce/extensions/edly_ecommerce_app/api/v1/tests/test_views.py]: nit: use single quotes

File: ecommerce/extensions/edly_ecommerce_app/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+backports.ssl-match-hostname==3.5.0.1
```
Comment (zubair-arbi) [inline ecommerce/extensions/edly_ecommerce_app/requirements.txt]: We can remove this requirements file. We will add one once we will make it a separate application.

Comment (asadmanzoor93) [reply ecommerce/extensions/edly_ecommerce_app/requirements.txt]: Feedback Implemented

File: ecommerce/extensions/edly_ecommerce_app/api/v1/tests/test_views.py (hunk 1 above)
Comment (asadmanzoor93) [reply ecommerce/extensions/edly_ecommerce_app/api/v1/tests/test_views.py]: Feedback Implemented

**Pull request merged**